playbooks/bibdata.yml

```diff
@@ -0,0 +1,26 @@
+---
+# by default this playbook runs in the staging environment
+# to run in qa, pass '-e runtime_env=qa'
+# to run in production, pass '-e runtime_env=production'
+- Build bibdata infrastructure
+  hosts: bibdata_{{ runtime_env | default('staging') }}
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../site_vars.yml
+    - ../group_vars/bibdata/vault.yml
+    - ../group_vars/bibdata/{{ runtime_env | default('staging') }}.yml
+  roles:
+    - role: roles/bibdata
+    - role: roles/bibdata_sqs_poller
+    - role: roles/sidekiq_worker
+      when: "'worker' in inventory_hostname"
```

Comment:
Cool when!

Reply:
I don't know if production needs the `samba_shares_root` var - so far I've only moved what existed before. But it certainly looks odd to have that var in staging and QA and not in production.

Reply:
Ah, it does need it. I missed this; it was in the production playbook originally.